GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -42,7 +42,7 @@ gdjs.RuntimeGamePixiRenderer.prototype.createStandardCanvas = function(canvasAre
 
     //Create the renderer and setup the rendering area
     //"transparent: true" is needed to avoid flickering issues on some mobile phones (see #566)
```

Comment:
You forgot to update the comment 😉 `"preserveDrawingBuffer: true" is needed to avoid [...]`